packages/babel-types/src/ast-types/generated/index.ts

```diff
@@ -855,6 +855,7 @@ export interface ImportDeclaration extends BaseNode {
   source: StringLiteral;
   assertions?: Array<ImportAttribute> | null;
   importKind?: "type" | "typeof" | "value" | null;
+  module?: boolean | null;
```

Comment:
Can we not use null?

Reply:
`t.ImportDeclaration` will assign `null` to `module` because it is an optional property.